tests/smoke/support/page/files/allFiles.ts

```diff
@@ -180,4 +193,61 @@ export class AllFilesPage {
     await page.click('#location-picker-btn-confirm')
     await page.goto(startUrl)
   }
+
+  async resourceExist({ 
+    name, 
+    version = false, 
+    notExist = false 
+  }: { 
+    name: string; 
+    version?: boolean; 
+    notExist?: boolean  
+  }): Promise<void> {
+    const { page } = this.actor
+    const startUrl = page.url()
+    const folderPaths = name.split('/')
+    const resouceName = folderPaths.pop()
+
+    if (folderPaths.length) {
+      await cta.files.navigateToFolder({ page: page, path: folderPaths.join('/') })
+    }
+    
+    if (notExist) {
+      const resourceExists = await cta.files.resourceExists({
+        page: page,
+        name: resouceName
+      })      
+      if (resourceExists) throw new Error(`selector was find: "${resouceName}"`)
+    } else if (version) {
+      await cta.files.sidebar.open({ page: page, resource: resouceName })
+      await cta.files.sidebar.openPanel({ page: page, name: 'versions' })
+      await page.waitForSelector('//div[@id="oc-file-versions-sidebar"]//tr[@class="file-row"]')
+    }
+    else {
+      await page.waitForSelector(`//*[@data-test-resource-name="${resouceName}"]`)
+    }
+     
+    await page.goto(startUrl)
+  }
+
+  async removeResourses({ resource }: { resource: string }): Promise<void> {
```

Comment:
there is a typo in resources, and i think its delete not remove

Reply:
renamed it